pages/common/fmt.md

```diff
@@ -0,0 +1,19 @@
+# fmt
+
+> Reformat a text file by joining its paragraphs and limiting the line  width to given number of characters (75 by default).
```

Comment:
```suggestion
> Reformat a text file by joining its paragraphs and limiting the line width to given number of characters (75 by default).
```

Typo.

Reply:
It's funny that I could have avoided this using `fmt`

Reply:
Hahaha :')